policy/policy.go

```diff
@@ -7,8 +7,11 @@ import (
 	"github.com/atomist-skills/go-skill/policy/policy_handler"
 )
 
+var VulnerabilityChangeEvent = "VulnerabilityChangeEvent"
```

Comment:
If the value is not supposed to change, could we define it as a `const` instead?